content/mentee-application.md

```diff
@@ -0,0 +1,83 @@
+---
+id: mentee-application
+title: Mentee Application Template
+---
+
+# Our Mentee Application Template
+
+Hello! Thank you for your interest in being a part of Galaxy Mentoring Network!
+
+**Kindly copy the content of this template and add paste it in a text editor so that you can have a draft before you tranfer your responses to the form.**
+
+# Section 1: Personal Information
+
+The information you entered in this section will be added to your profile on our website. Be free to let's know who you are!
+
+1. First Name
+2. Last Name
+3. Email
+4. Gitter/Matrix ID
+5. State/Country
+6. Pronouns
+7. Affiliations
+8. Age (in ranges:) ***Choose one option***
+    20-30
+    30-40
+    40-50
+    50 - 60
+    60-70
+    70+
+    Prefer not to say
```

Comment:
Could you put as a list?

Reply:
It looks strange: 
![image](https://user-images.githubusercontent.com/1842467/153849545-31f32718-910e-40fc-87ea-399f8023ba4c.png)